src/gui/search/searchwidget.cpp

```diff
@@ -83,6 +87,7 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     , m_isNewQueryString(false)
 {
     m_ui->setupUi(this);
+    m_ui->tabWidget->tabBar()->installEventFilter(this);
```

Comment:
Is it a good idea to install this class as the event filter?
Take a look at `class WheelEventEater`, I think that is a better way.

Reply:
Whichever class has the `eventFilter` function needs to be installed. As for which class it gets installed too, that has to be the tab bar because the coordinates are relative. I first tried installing it to the search widget, but all of the coordinates for the mouse events were offset, so `tabAt` would always return `-1`.